hw/ip/edn/data/edn.hjson

```diff
@@ -371,6 +371,15 @@
                 This bit will stay set until the next reset.
                 '''
         }
+        { bits: "2",
+          name: "SFIFO_OUTPUT_ERR",
+          desc: '''
+                This bit will be set to one when an error has been detected for the
+                output FIFO. The type of error is reflected in the type status
+                bits (bits 28 through 30 of this register).
+                This bit will stay set until the next reset.
```

Comment:
This is also a fatal error right?
Would you mind adding that to the description?

Reply:
Normally would not mind, but any error in this register should be considered fatal, as there is another register that has "recoverable error" in the title. This register is replicated across three blocks for consistency, so they should all change if we end of changing just this one. What do you prefer?

Reply:
Oh! Thanks Mark. Totally make sense :)  That is a good point and I was actually a little bit confused on that:
When I tested `err_code_test` register:
Error: 0,1,2,20,21,22 will trigger fatal alert.
Error: 28, 29, 30 won't trigger fatal alert.

So I thought only the ones that specified "it will trigger fatal alert" will trigger :)

Reply:
ah yes.